examples/analog_in/ai_voltage_waveform_cont_acq_with_event.py

```diff
@@ -0,0 +1,32 @@
+﻿"""Example of analog input voltage acquisition.
+
+This example demonstrates how to continuously acquire buffered voltage measurements using
+a DAQmx device. It uses software events to ensure that the program does not become
+unresponsive while waiting for samples to become available.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+with nidaqmx.Task() as task:
+    samples = []
+
+    def callback(task_handle, every_n_samples_event_type, number_of_samples, callback_data):
+        """Callback function for reading singals."""
+        print("Every N Samples callback invoked.")
+
+        samples.extend(task.read(number_of_samples_per_channel=100))
```

Comment:
this buffer will get very large, very quick. Just print here instead? I like accumulating the total number of samples, though, so just have a variable for that and print at the end

Reply:
yes, see `ContAcq-IntClk-EveryNSamplesEvent.c`